docs/extensions/tutorials/_posts/1970-01-01-SettingLocalEnvironment.md

```diff
@@ -21,114 +21,95 @@ Before setting up a local environment for building Shoutem extensions, set up Re
-Running `News app` app on `iPhone 6` simulator...
-...
+$ cd Restaurants
+$ react-native run-ios
 ```
 
-> #### Note
-> The name of your app may be different if you decided to rename it in the Builder, simply make sure to select the app you installed your Hello World extension into.
+Now you can develop your extension much faster because you can see the changes you make to your extension in real time, exactly like a regular React Native app.
 
-When the app was run, the code from your linked extension was taken locally and other extensions were fetched from Shoutem server. Running the app will take some time (around 3-4mins), but now you can develop your extension much faster because you can see the changes you make to your extension in real time, exactly like a regular React Native app.
-
-Lets see how this works. Change something inside your extension, for example you could add another line of `<Text>`:
+Lets see how this works. Change something inside your extension, for example you could add another line of `<Text>` to the List screen:
 
 ```javascript{5}
   render() {
     return (
       <View style={styles.container}>
-        <Text style={styles.text}>Hello World!</Text>
+        <Text style={styles.text}>Lets eat!</Text>
         <Text style={styles.text}>This is my first extension!</Text>
       </View>
     );
   }
 ```
 
-Save the changes. Now you don't need to do `shoutem push` anymore since you linked this extension to your local environment. You can just reload the app:
-
+Save the changes. Now you don't need to do `shoutem push` and wait for the Builder to load the changes, you can just reload the app:
 - On simulator:
   - iOS: press `Command ⌘` + `R`
   - Android: press `R` twice
-- On physical device: shake your device to open up **Developer Menu** and select **Reload**
+- Using `shoutem run`: shake your device to open up **Developer Menu** and select **Reload**
 
 If you enable automatic reloading, the previous step is unnecessary. Having local environment set, you can also debug your extension. Follow the React Native guide on [debugging](https://facebook.github.io/react-native/docs/debugging.html) where you can find out how automatic reloading works as well.
 
-To see which extensions are linked, as well as which developer is signed in, run:
+### Managing your app's Configuration
 
-```ShellSession
-$ shoutem show
-Linked directories:
-  /Path/to/hello-world
-Registered as `{{ site.example.devName }}`.
-```
+This section will cover the uses of the configuration script and associated commands.
 
-To unlink an extension, locate to that folder and do:
+#### shoutem configure
 
 ```ShellSession
-$ shoutem unlink
-Unlink successful. Please, kill the packager before running the app.
+$ shoutem configure
+...
 ```
 
-... or just delete the extension.
+You use it when:
+    - creating and installing a new extension locally using `shoutem init`, `shoutem push` and `shoutem install`
```

Comment:
You `should` use it when (see what Ivo wrote in the new CLI reference, also in the PR).